### PR TITLE
Deploy to mainnet

### PR DESCRIPTION
**Pull request by xavier506**
merge staging into master

### CODE REVIEW

**Review by JustinCast — APPROVED** [review]
LGTM

File: services/frontend/.env [hunk 1]
```diff
@@ -1,6 +1,6 @@
-REACT_APP_GRAPHQL_HTTP_URL=https://graphql-jungle.eosrate.io/v1alpha1/graphql
+REACT_APP_GRAPHQL_HTTP_URL=http://localhost:8088/v1/graphql
```
Comment (JustinCast) [inline services/frontend/.env:1]: @xavier506 it is necessary to let the previous values, isn't it?
I think I forgot to remove this

**Review by JustinCast — COMMENTED** [review]
image
Do we need this config?
I'm asking since we deprecated the demux service

File: services/frontend/.env (hunk 1 above)
Comment (xavier506) [reply services/frontend/.env:1]: we should have `.env` files in the gitignore and use github secrets to manage env vars passed along in the github workflows

**Comment by xavier506** [comment]
> image
> Do we need this config?
> I'm asking since we deprecated the demux service

Yeah we aren't using demux but we can still keep the code around, docker compose is used for local environments , \  maybe comment it ?

File: services/frontend/.env (hunk 1 above)
Comment (JustinCast) [reply services/frontend/.env:1]: @xavier506 some time ago I added .env files to .gitignore and we had some problems in the pipeline. For now I think it is better to just replace the values with the previous ones

**Review by kuronosec — APPROVED** [review]
LGTM

**Pull request merged**